apps/internal/oauth/ops/accesstokens/accesstokens.go

```diff
@@ -142,7 +151,7 @@ func (c *Credential) JWT(ctx context.Context, authParams authority.AuthParams) (
 // https://tools.ietf.org/html/rfc7517#section-4.8
 func thumbprint(cert *x509.Certificate) []byte {
```

Comment:
This is all fine for AAD. But for ADFS and dSTS, we should continue to use SHA-1.

Reply:
Updating on it